Jenkinsfile

```diff
@@ -9,7 +9,7 @@ buildPlugin(
  // See also https://github.com/jenkins-infra/pipeline-library/pull/577
  //          https://github.com/jenkins-infra/pipeline-library/pull/522
  //          https://github.com/jenkins-infra/pipeline-library/pull/635
- useArtifactCachingProxy: false,
+ useArtifactCachingProxy: true,
```

Comment:
The whole L7-L12 can be deleted, let me know if you want me to update the PR accordingly.

Reply:
Thanks, did so now!